docs/dev.md

```diff
@@ -6,6 +6,8 @@ Start a Liberty server in dev mode. This goal also invokes the `create`, `instal
 
 N.B. starting in the tech preview 3.5.2-SNAPSHOT, dev mode also invokes `generate-features`. See [generate-features](generate-features.md) for details.
 
+Additionally, starting in 3.5.2-SNAPSHOT, [resource variable filtering](https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html) and [WAR overlays](https://maven.apache.org/plugins/maven-war-plugin/overlays.html) are supported for loose WAR applications. This is done by automatically detecting appropriate Maven WAR plugin configuration (v3.3.1+ recommended) and calling the WAR plugin's `exploded` goal and the Maven Resource plugin's `resource` goal when appropriate. 
```

Comment:
I think it's 3.3.2 that's recommended.   Maybe 3.3.1 has the setting but not the last bit of function I added in 3.3.2.

Reply:
Went back and read UFO.   If we'd recommend anything it's probably 3.3.2 but it's complicated.   

We're trying to defer to the plugin and accepting that if someone has an issue with it we might have to walk them through.    Not super likely a user's going to benefit from reading this in the doc, but the doc is useful to us as maintainers to remember our intent.    We don't want to bog down with something long. 

We're eventually going to maybe want to do something similar to the overlay behavior.

What about adding? : 

> This is done by automatically detecting appropriate Maven WAR plugin configuration and calling the WAR plugin's `exploded` goal and the Maven Resource plugin's `resource` goal when appropriate.   Behavior for updating/deleting resources is delegated via the Maven WAR plugin configuration, including the `outdatedCheckPath` parameter enhanced in plugin version 3.3.2.

Reply:
That sounds good to me